### PR TITLE
Feature/checker cli

### PR DESCRIPTION
**Pull request by agduncan94**
This implements the basics for checker cli.

### CODE REVIEW

**Comment by agduncan94** [comment]
Tests will be added in an upcoming PR.

File: dockstore-client/src/main/java/io/dockstore/client/cli/CheckerClient.java [hunk 1]
```diff
@@ -0,0 +1,446 @@
...
+                    errorMessage("The entry has no checker workflow.",
+                        Client.CLIENT_ERROR);
+                } else {
+                    checkerWorkflow = workflowsApi.getWorkflow(entry.getCheckerId());
+                }
+            }
+
+            // Call parent launcher
+            if (entry != null && checkerWorkflow != null) {
+                // Readd entry path to call, but with checker workflow
```
Comment (garyluu) [inline dockstore-client/src/main/java/io/dockstore/client/cli/CheckerClient.java]: typo

File: dockstore-client/src/main/java/io/dockstore/client/cli/CheckerClient.java [hunk 2]
```diff
@@ -0,0 +1,446 @@
...
+                launch(args);
+            }
+        }
+    }
+
+    private void testParameterChecker(List<String> args) {
+        if (containsHelpRequest(args) || args.isEmpty()) {
+            testParameterHelp();
+        } else {
+            // Retrieve arguments
```
Comment (garyluu) [inline dockstore-client/src/main/java/io/dockstore/client/cli/CheckerClient.java]: 'Retrieve arguments' up to 'Get checker workflow` section eerily similar to updateVersionChecker

File: dockstore-client/src/main/java/io/dockstore/client/cli/Client.java [hunk 3]
```diff
@@ -825,4 +830,14 @@ public ToolClient getToolClient() {
...
         return workflowClient;
     }
+
+    /**
+     * Setup method called by Consonance
+     *
+     * @return
+     */
+    @SuppressWarnings("WeakerAccess")
+    public CheckerClient getCheckerClient() {
```
Comment (denis-yuen) [inline dockstore-client/src/main/java/io/dockstore/client/cli/Client.java]: Seems odd. Maybe this is missing a `@Override` to explain this?

File: dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java [hunk 4]
```diff
@@ -57,7 +58,20 @@
 @Inheritance(strategy = InheritanceType.TABLE_PER_CLASS)
 @SuppressWarnings("checkstyle:magicnumber")
 // TODO: Replace this with JPA when possible
-@NamedNativeQuery(name = "Entry.getEntryById", query = "SELECT 'tool' as type, id from tool where id = :id union select 'workflow' as type, id from workflow where id = :id")
+@NamedNativeQueries({
```
Comment (denis-yuen) [inline dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java]: I think we've created a monster

**Pull request merged**